mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -47,6 +47,15 @@ namespace System.Drawing
 	/// GDI+ API Functions
 	/// </summary>
 	internal static class GDIPlus {
+
+#if NETSTANDARD1_6
+		public const CharSet GdiPlusCharSet = CharSet.Unicode;
+		public const UnmanagedType GdiPlusCustomMarshaler = UnmanagedType.Interface;
```

Comment:
does using UnmanagedType.Interface actually work in NS16/netcore or are those methods just never hit?